src/context/AuthContext/AuthContext.tsx

```diff
@@ -0,0 +1,29 @@
+// import React from 'react'
+
+import { jwtDecode } from 'jwt-decode';
+import { createContext, useEffect, useState } from 'react';
+
+export const AuthContext = createContext(null);
+export default function AuthContextProvider({ children }) {
+  const [loginData, setLoginData] = useState(null);
+
+  const saveLoginData = () => {
+    const encodeToken = localStorage.getItem('token');
+    const decodeToken = jwtDecode(encodeToken);
```

Comment:
I suggest using get current user api to get the user data including the user's image instead of relying on a third party package.